opslevel/datasource_opslevel_domain.go

```diff
@@ -89,59 +89,15 @@ func (d *DomainDataSource) Read(ctx context.Context, req datasource.ReadRequest,
 	}
 
 	data.Id = types.StringValue(string(domain.Id))
+	domainAliases, aliasesDiag := types.ListValueFrom(ctx, types.StringType, domain.Aliases)
+	resp.Diagnostics.Append(aliasesDiag...)
+
+	data.Aliases = domainAliases
+	data.Name = types.StringValue(string(domain.Name))
+	data.Description = types.StringValue(string(domain.Description))
+	data.Owner = types.StringValue(string(domain.Owner.Id()))
 	tflog.Trace(ctx, "read an OpsLevel Domain data source")
 
 	// Save data into Terraform state
 	resp.Diagnostics.Append(resp.State.Set(ctx, &data)...)
 }
-
```

Comment:
😢 - GITHUB YOU CAN DO BETTER THEN THIS.  Might pull this up in beyound compare or P4Diff because this his horrible.